content/_data/changelogs/lts.yml

```diff
@@ -140,6 +140,77 @@
       message: Do not fail to write a log file just because something deleted the parent directory.
       issue: 16634
       pull: 2738
+- version: "2.46.1"
+  date: 2017-04-19
+  lts_predecessor: "2.32.3"
+  lts_baseline: "2.46"
+  lts_changes:
+    - type: rfe
+      message: Update the SSHD module from 1.7 to 1.8. The change disables obsolete Ciphers: AES128CBC, TripleDESCBC, and BlowfishCBC.
```

Comment:
According to POM XML, the version is 1.9. This entry comes from the 2.34, and it conflicts with the entrues for 2.37:

* Update SSHD Core from 0.8.0 to 0.14.0. (pull 2662)
* SSHD Module: Handshake was failing (wrong shared secret) 1 out of 256 times due to SSHD-330. (issue 40362)